From c54436e38a76b691919145bc4814622cc8d71141 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?R=C3=A9mi=20Bernon?= <rbernon@codeweavers.com>
Date: Tue, 12 Nov 2019 16:11:55 +0100
Subject: [PATCH 08/16] user32/tests: Wait indefinitely for the child process
 to notify.

We could create the process, so now it should eventually signal the
event unless there's something very wrong going on. This wait sometimes
times out on the testbot, this makes it infinite.
---
 dlls/user32/tests/msg.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dlls/user32/tests/msg.c b/dlls/user32/tests/msg.c
index 11111111111..11111111111 100644
--- a/dlls/user32/tests/msg.c
+++ b/dlls/user32/tests/msg.c
@@ -17900,7 +17900,7 @@ static void test_WaitForInputIdle( char *argv0 )
         ok( ret, "CreateProcess '%s' failed err %lu.\n", path, GetLastError() );
         if (ret)
         {
-            ret = WaitForSingleObject( start_event, 5000 );
+            ret = WaitForSingleObject( start_event, INFINITE );
             ok( ret == WAIT_OBJECT_0, "%u: WaitForSingleObject failed\n", i );
             if (ret == WAIT_OBJECT_0)
             {
-- 
2.39.0

